app/controllers/general.php

```diff
@@ -90,15 +90,15 @@
                 if (!empty($envDomain) && $envDomain !== 'localhost') {
                     $mainDomain = $envDomain;
                 } else {
-                    $domainDocument = $dbForConsole->findOne('domains', [], 0, ['_id'], ['ASC']);
+                    $domainDocument = $dbForConsole->findOne('domains', [Query::orderAsc('_id')]);
```

Comment:
Don't we need to include the offset here ? or is the default offset 0 ? 
I see that we have explicitly used `Query::offset(0)` in some places

![Screenshot 2022-08-13 at 2 22 42 PM](https://user-images.githubusercontent.com/20852629/184476576-009038ad-0545-49fc-a55e-6bab244b08c3.png)
 
 Shall we choose one of the patterns and stay consistent ? 
 
 
 Either use the default offset 0 param everywhere or explicitly specify it with `Query::offset(0)`